### PR TITLE
fix: adjust width of week chart and hours

### PR DESCRIPTION
**Pull request by GusBaamonde**
- Hover effect adjustment on each circle

week-hours-100

### CODE REVIEW

File: src/components/Reports/ReportsHoursVisits/ReportsHoursVisits.styles.js [hunk 1]
```diff
@@ -68,7 +67,7 @@ export const Circle = styled.span`
   width: 11px;
   height: 11px;
   background-color: ${colors.purple1};
-  box-shadow: 0p 0p 0 30px rgba(255, 255, 255, 0);
+  box-shadow: 0px 0px 0 30px rgba(255, 255, 255, 0);
```
Comment (andresmoschini) [inline src/components/Reports/ReportsHoursVisits/ReportsHoursVisits.styles.js]: Maybe:

```suggestion
  box-shadow: 0 0 0 30px rgba(255, 255, 255, 0);
```

File: src/components/Reports/ReportsHoursVisits/ReportsHoursVisits.styles.js [hunk 2]
```diff
@@ -86,19 +85,19 @@ export const Circle = styled.span`
       height: 31px;
     `}
 
-   &:hover {
+    &:hover {
```
Comment (andresmoschini) [inline src/components/Reports/ReportsHoursVisits/ReportsHoursVisits.styles.js]: 👍

**Comment by fcoronelmakingsense** [comment]
i have only a question when happens is resize the the screen? it's broken or fluid?

**Comment by GusBaamonde** [comment]
> i have only a question when happens is resize the the screen? it's broken or fluid?

It is fluid up to 1024, but when we have the grid we will adjust what is missing

**Comment by GusBaamonde** [comment]
test this please

**Pull request merged**